src/localization/successor_manager/node.h

```diff
@@ -44,11 +44,11 @@ class Node {
   void print() const;
 
   /** WARNING: use only for the priority queue.
- * Comparison is based on the accumulated cost **/
+   * Comparison is based on the accumulated cost **/
   bool operator<(const Node &rhs) const { return this->accCost > rhs.accCost; }
 };
 
-#define SOURCE_NODE Node(-1, 0, 0.0)
+#define SOURCE_NODE Node(/*queryId=*/-1, /*refId=*/0, /*idvCost=*/0.0)
```

Comment:
Why do you need a macros here? You could instead just create a constant:
```c++
static const Node kConstNode{/*queryId=*/-1, /*refId=*/0, /*idvCost=*/0.0};
```
Or am I missing something here?